reframe/core/buildsystems.py

```diff
@@ -897,6 +897,29 @@ def prepare_cmds(self):
         return cmds
 
 
+class CustomBuild(BuildSystem):
```

Comment:
I would rename this to `Custom` since the user is going to use:
```
build_system = 'Custom'
```

Reply:
Yeah, I though about this, but `build_system = 'CustomBuild'` is not bad either, is it? In the sense that you are doing "custom build" after all.

Reply:
I am fine with both.